### ISSUE TITLE
Cannot store buckets in crates

### ISSUE THREAD

**Issue by TomyLobo** [Funwayguy/BarrelsDrumsStorageAndMore#27]
As it says in the title.
Looking at the code, this is probably true for all fluid handlers

**Comment by Funwayguy** [comment]
This is intended. It applies to any container based item (item/fluid/energy) to prevent nesting exploits

**Comment by TomyLobo** [comment]
how would you nest another container in a fluid handler unless it is also an item handler?

**Comment by Funwayguy** [comment]
In the case of fluids its more so to prevent people from using crates for fluid storage instead of the purpose made barrels or stacking barrels from other mods into crates (that'd be some OP capacity right there)

**Comment by TomyLobo** [comment]
How about excluding buckets?

My use case was actually already a workaround.
I'm using integrated dynamics+integrated crafting to set up some autocrafting.
One recipe needed a bucket of liquified redstone (the actual name escapes me atm).
We have Forestry in our modpack so I tried using the Bottler for it.
However, that thing exports fluids into adjacent fluid handlers (including back into the ID crafting interface).
So I thought I'd put in an intermediate storage so that wouldn't happen.
Not that easy to find a general-purpose storage which supports liquids *and* items.
Shipping Container came to mind, since you can put drums and crates into it.
However, for some reason it wouldn't accept the buckets at all, so I was stuck and opened this issue asking for that restriction to be removed.

However, I have since found another workaround using a sub-network.

**Comment by PrincessRTFM** [comment]
I'm not sure I understand the criteria for "can I put this into a shipping crate"... I found this issue because I was looking for a way to, essentially, use the shipping container to hold something that will itself hold my items. My first thought was Thermal Expansion's strongboxes or satchels. Those are forbidden, but TE's energy cells aren't. At present, buckets work fine, and so do the handful of modded tanks I tested. But when I tried using a vanilla shulker box, it not only was accepted, it also started storing items inside.

What decides whether an item with its own inventory is allowed into the shipping crates?
